api-server/stack-test-suite/tests/v1/block_reward.rs

```diff
@@ -169,7 +172,12 @@ async fn has_reward(#[case] seed: Seed) {
 
                     _ = tx.send((
                         block_index.block_id().to_hash().encode_hex::<String>(),
-                        json!(block.block_reward().outputs().iter().clone().collect::<Vec<_>>()),
+                        json!(block
+                            .block_reward()
+                            .outputs()
+                            .iter()
+                            .map(|out| txoutput_to_json(out, &chain_config))
+                            .collect::<Vec<_>>()),
```

Comment:
It's a test so no big deal by anyway: Is the `json!` macro needed here? I don't see any json-specific syntax, just a rust expression. Also `serde_json::Value` implements `from_iter` so you should be able  to `.collect::<serde_jsno::Value>()` if I'm not mistaken.

Reply:
fixed with the collect to serde_json::Value